app/views/notes/index.html.erb

```diff
@@ -1,6 +1,7 @@
 <h1>My Notes</h1>
 <%= simple_form_for @new_note, html: {id: "note_form"}, remote: true do |f| %>
   <%= f.input :title, label: false, placeholder: "Enter a Note" %>
+  <%= f.input :user_id, as: :hidden %>
```

Comment:
Remove this.
Instead, the create action should be, `current_user.tasks.create(task_params)`